portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyBlogTests.testcase

```diff
@@ -0,0 +1,60 @@
+@component-name = "portal-lima"
+definition {
+
+	property testray.main.component.name = "Blogs";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+
+		task ("Given the Blogs Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Blogs Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = 1,
+				groupName = "Guest",
+				layoutName = "Blogs Page",
+				widgetName = "Blogs");
+
+			Navigator.gotoPage(pageName = "Blogs Page");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Can Create a Blog"
+	@priority = 5
+	test CanCreateBlog {
+		task ("When the user adds a Blog Entry") {
+			Click(
+				key_text = "New Entry",
+				locator1 = "Button#ANY");
+
+			WaitForSPARefresh();
+
+			BlogsEntry.addTitle(entryTitle = "Blog Entry Title");
+
+			BlogsEntry.addContent(entryContent = "Blog Entry Content");
+
+			PortletEntry.publish();
```

Comment:
Hey, can you take a look at this macro: `Blogs.addEntry`? I believe it does the same thing you are doing in lines 10 - 50
When you open the macro you will see it can receive a lot of arguments, but you don't have to use all of them, I believe you just have to use the `entryContent` and `entryTitle`

Reply:
Thank you!